taxi/models.py

```diff
@@ -3,6 +3,7 @@
 
 
 class Manufacturer(models.Model):
+    objects = object
```

Comment:
The `objects` attribute is incorrectly assigned as `object`. It should be `models.Manager()` or simply removed, as Django automatically provides a default manager named `objects`.